lib/services/devices/registrationUtils.js

```diff
@@ -136,7 +136,6 @@ function sendUnregistrationsNgsi2(deviceData, callback) {
     const options = {
         url: cbHost + '/v2/registrations/' + deviceData.registrationId,
         method: 'DELETE',
-        json: true,
```

Comment:
Yes, removal of `json:true` should be sufficient - but I guess the old `request` library just ignored this anyway so the line was never necessary - same for the other instances.